src/odemis/driver/test/actuator_test.py

```diff
@@ -548,6 +548,49 @@ def test_metadata(self):
 
         testing.assert_pos_almost_equal(stage.speed.value, {"x": 10e-6, "y": 20e-6})
 
+    def test_get_axes_range(self):
+        """
+        Test the range calculation in ConvertStage
+        """
+        # Test based on given range of Zeiss stage
+        dependency = simulated.Stage("stage", "test", axes=["y", "m"],
+                                     ranges={"y": [0.e-3, 100.e-3], "m": [0.e-3, 13.e-3]})
+
+        # Transform into Meteor stage , rotated by 26 degrees --> 0.4537856055185 rad
+        tilt = math.radians(26)
+        stage = ConvertStage("conv", "stage", {"orig": dependency},
+                             axes=["y", "m"], rotation=tilt)
+        first_value = 2e-03
+        second_value = 0
```

Comment:
Could you try to move to some places which are impossible within the "dependency" range. For instance, a negative value in x?

Reply:
No, that's still too easy, because first_value and second_value (which you could just call x_pos and y_pos) are also "legal" values as a & b in the dependency. Please use values which would be forbidden as a & b *but allowed* as x & y. For instance something like -2mm/10mm. (You need to check these values, I don't know if that's correct!).

Reply:
> Could you try to move to some places which are impossible within the "dependency" range. For instance, a negative value in x?

I performed other test to check movements outside the a/b range. It starts on line 568.

Reply:
Added the test 

> No, that's still too easy, because first_value and second_value (which you could just call x_pos and y_pos) are also "legal" values as a & b in the dependency. Please use values which would be forbidden as a & b _but allowed_ as x & y. For instance something like -2mm/10mm. (You need to check these values, I don't know if that's correct!).